tensorrt_llm/_torch/models/modeling_nemotron_nano.py

```
@@ -1127,26 +1168,529 @@ def get_num_tokens_per_video(
+            # `merge_evs_mm_embeds` rewrites it at forward time.
+            tokens_per_frame = self._compute_token_numbers_per_video([video_size])[0]
+            frame_seps = self._get_frame_separators([video_size], [metadata])[0]
+
+            expansion: List[int] = []
+            evs_expansion: Optional[List[int]] = [] if evs_enabled else None
+            if self._add_video_prefix:
+                prefix_ids = self.tokenizer.encode("This is a video:\n", add_special_tokens=False)
+                expansion.extend(prefix_ids)
+                if evs_expansion is not None:
+                    evs_expansion.extend(prefix_ids)
+            for frame_sep, num_tokens in zip(frame_seps, tokens_per_frame):
```

Ruff (0.15.10)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)